tests/org.jboss.tools.examples.ui.bot.test/src/org/jboss/tools/examples/ui/bot/test/integration/AbstractImportQuickstartsTest.java

```diff
@@ -215,10 +215,12 @@ public boolean accept(File arg0) {
 //						"Picketlink was skipped due to JBIDE-18497");
 //				continue;
 //			}
-			if(specificQuickstarts.size()==0||specificQuickstarts.contains(f.getName())){
-				System.out.println("PROCESSING " + f.getAbsolutePath());
-				Quickstart qstart = new Quickstart(f.getName(), f.getAbsolutePath());
-				resultList.add(qstart);
+			if(!f.getPath().contains("/.")){
+				if(specificQuickstarts.size()==0||specificQuickstarts.contains(f.getName())){
+					System.out.println("PROCESSING " + f.getAbsolutePath());
```

Comment:
I know this is artifact of my previous work (sorry for that), but would you mind to remove this sysout as part of this PR? We're trying to have no System.out in our code. This one can be just deleted or replaced by logger.info or something like that.